conda/core/subdir_data.py

```diff
@@ -808,10 +603,24 @@ def cache_fn_url(url, repodata_fn=REPODATA_FN):
     return f"{md5.hexdigest()[:8]}.json"
 
 
-def add_http_value_to_dict(resp, http_key, d, dict_key):
-    value = resp.headers.get(http_key)
-    if value:
-        d[dict_key] = value
+def fetch_repodata_remote_request(url, etag, mod_stamp, repodata_fn=REPODATA_FN):
+    # this function should no longer be used by conda but is kept for API stability
+    warnings.warn(
+        "fetch_repodata_remote_request",
+        PendingDeprecationWarning,
```

Comment:
todo write the automatic deprecation until version x.y function

Reply:
Oh wait, this warning needs to be updated to be end-user compatible :D